spec/requests/admin/users_spec.rb

```diff
@@ -125,9 +125,9 @@
         expect( response      ).to     have_http_status :ok
         expect( response.body ).to     have_title I18n.t( 'admin.users.index.title' ).titlecase
         expect( response.body ).to     have_css '.alert-success', text: I18n.t( 'admin.users.destroy.success' )
-        expect( response.body ).to     include u1.username
-        expect( response.body ).not_to include u2.username
-        expect( response.body ).to     include u3.username
+        expect( response.body ).to     have_css 'td', text: u1.username
+        expect( response.body ).not_to have_css 'td', text: u2.username
+        expect( response.body ).to     have_css 'td', text: u3.username
```

Comment:
If it makes the intent clearer and the test more readable, consider using Capybara's `have_table` matcher along with one of its `cols`,`rows`,`with_cols`,`with_rows` options to replace lines 128-130. 

- https://rubydoc.info/github/teamcapybara/capybara/master/Capybara%2FRSpecMatchers:have_table
- Options documented at https://rubydoc.info/github/teamcapybara/capybara/master/Capybara%2FNode%2FMatchers:has_table%3F